src/components/common/input/DisabledInput.tsx

```
@@ -1,31 +1,31 @@
-      <InputGroup>
-        <Input id="email" placeholder={placeholder} disabled />
-        <InputRightElement children={<CheckIcon color="green.500" />} />
-      </InputGroup>
-    </FormControl>
-  );
+	return (
+		<FormControl>
+			<FormLabel htmlFor="email">{label}</FormLabel>
+			<InputGroup>
+				<Input id="email" placeholder={placeholder} disabled />
+				<InputRightElement children={<CheckIcon color="green.500" />} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)